user_deprovision/user_deprovision.py

```diff
@@ -431,8 +432,14 @@ def delete_user(access_token, user_email, from_email, prompt_del, auto_resolve,
 
 
 def main(args):
-    n_u = len(args.user_emails)
-    print("%d users to delete: %s"%(n_u, ', '.join(args.user_emails)))
+    email_list = []
+    for (i, item) in enumerate(csv.reader(args.user_csv)):
```

Comment:
For clarity, can we use 'row' instead of 'item' so it's clearer what we're grabbing here. 
I would also recommend `email = row[0]` for the same reason.